src/Microsoft.OData.Core/Parameterized.Microsoft.OData.Core.cs

```diff
@@ -1647,6 +1647,14 @@ internal static string ODataBatchReaderStreamBuffer_BoundaryLineSecurityLimitRea
             return Microsoft.OData.TextRes.GetString(Microsoft.OData.TextRes.ODataBatchReaderStreamBuffer_BoundaryLineSecurityLimitReached, p0);
         }
 
+        /// <summary>
+        /// A string like "Duplicate property name '{0}' for message in batch."

```

Comment:
>message [](http://example.com/codeflow?start=61&length=7)

suggest: "request" rather than "message"

Reply:
Changed naming of the exceptions from "message" to "request"